README.md

```diff
@@ -26,6 +28,12 @@ Steps inlcude:
 
 ![metro map](docs/images/metro_map.png)
 
+UMI processing (only for rows with `umi_type`):
+- Extract UMI from read sequence (`seq`) or read name (`readname`)
+- Group reads by UMI (fgbio GroupReadsByUmi)
+- Call molecular consensus (fgbio CallMolecularConsensusReads) and filter (fgbio FilterConsensusReads)
+- Re-align filtered consensus reads with BWA-MEM (`-Y`), then sort/index
+
```

Comment:
Can you guys also add this to the metro map if you have some time left? You can find tutorials for this here: https://nf-co.re/docs/guidelines/graphic_design/overview